src/main/java/seedu/address/model/person/NameContainsKeywordsPredicate.java

```diff
@@ -16,7 +16,7 @@ public NameContainsKeywordsPredicate(List<String> keywords) {
     @Override
     public boolean test(Person person) {
         return keywords.stream()
-                .anyMatch(keyword -> person.getName().fullName.toLowerCase().contains(keyword.toLowerCase()));
+                .anyMatch(keyword -> person.getName().containsKeyword(keyword.toLowerCase()));
```

Comment:
The same goes for this, through the validation check, this would not be necessary.